src/main/java/data/SheetsServiceUtil.java

```diff
@@ -16,17 +16,23 @@
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.sheets.v4.Sheets;
 import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.googleapis.extensions.appengine.auth.oauth2.AppIdentityCredential;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.sheets.v4.Sheets;
+
 
 /** A class to create a Sheets instance. */
 public class SheetsServiceUtil {
 
   public static Sheets getSheetsService() throws IOException, GeneralSecurityException {
-    AppIdentityCredential credential = SheetsAuthUtil.authorize();
-
-    return new Sheets.Builder(new UrlFetchTransport(), new JacksonFactory(), credential)
-      .build();
+    // Create a spreadsheet by using the OAuth tokens.
+    return null;
```

Comment:
I'd rather we remove this file all together, than replace the impl with something that returns null.

Reply:
I might want to add functionality to this file once the OAuth is set up. I don't want to have to delete it and then recreate it again.